09a_Custom_Coadd.ipynb

```diff
@@ -0,0 +1,1386 @@
+    "\n",
+    "**The `from_uri` method.**\n",
+    "    \n",
+    "The following method for creating the `assembleCoaddPipeline` task in a notebook -- using the `from_uri` function and passing a file path -- is not intuitive, but is preferred because it gives the user all of the configuration settings for the instrument automatically, and because it most closely replicates the command-line syntax.\n",
+    "\n",
+    "**The \"yaml\" file.**\n",
+    "    \n",
+    "A \"yaml\" file is a human-readable data-serialization language.\n",
+    "It is commonly used for configuration files and in applications where data is being stored or transmitted. \n",
+    "    \n",
+    "The other tasks available are listed in the yaml file. \n",
+    "To see the other tasks, first open a new terminal (click the blue + button at upper left and then select terminal).\n",
+    "Then create a Rubin Observatory environment, navigate to the DRP.yaml file, and view its contents with:\n",
+    "> `setup lsst_distrib` <br>\n",
+    "> `cd ${PIPE_TASKS_DIR}/pipelines/`<br>\n",
+    "> `more DRP.yaml`\n",
+    "\n",
+    "When attempting to use `more` on the DRP.yaml file, a redirect to its true location might be returned. If so, follow the path given.\n",
+    "    \n",
+    "> **Notice:** Below we are selecting to use the yaml file with DP0-specific configurations. This is likely to change for future data previews.\n",
+    "   "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "881e2e65-dad7-426e-a631-74bacaad209e",
+   "metadata": {},
+   "source": [
+    "First create `my_uri`, a string that defines which parts of the Data Release Pipeline (DRP) will be included."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "63d2e0f9-8d04-4045-b182-f112ee7c5430",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "yaml_file = '$DRP_PIPE_DIR/pipelines/LSSTCam-imSim/DRP-DP0.2.yaml'\n",
+    "steps = 'makeWarp,assembleCoadd'\n",
+    "my_uri = yaml_file + '#' + steps\n",
+    "print(my_uri)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5a35cd5b-5a71-449f-a15a-4694c84ccb51",
+   "metadata": {},
+   "source": [
+    "Next create a pipeline including just `makeWarp` and `assembleCoadd` steps, and call it `assembleCoaddPipeline`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "31df8776-0574-4d69-804e-4ddc7e78dde1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "assembleCoaddPipeline = Pipeline.from_uri(my_uri)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "de7ebc47-f65e-4c46-b9c4-954248c32478",
+   "metadata": {},
+   "source": [
+    "## 3.4. Configure the pipeline\n",
+    "\n",
+    "Configurations can be set using `addConfigOverride(<taskName>, <configName>, <configValue>)`.\n",
+    "\n",
+    "There is only one configuration that must be set, and it is to clarify to the pipeline that it _does not_ need to redo a final image characterization step.\n",
```

Comment:
In fact, having reached the end of the notebook where it prints the config items to the screen, I'm thinking it could be good to set one or more other configs. For example, what if we set "coaddName" to "window1" (instead of the default: "deep")?

Reply:
I don't disagree with changing one of the configs, but I don't think it should be 'coaddName'. I'm pretty sure that the 'coaddName' here sets the datasetType of the output, so by changing the name, you would add a new datasetType to the database, which we don't want users to do casually.

Reply:
It is kind of from thin air. The only reason that config is needed to be reset is to deal with an error message. I added "in general, users of this tutorial do not need to worry about other configuration changes".